CreeDictionary/search_quality/sample.csv

```diff
@@ -1,70 +1,384 @@
+named,isiyihkâcikâtêw,isiyihkâsow,,,
+names,isiyihkâtâw,isiyihkâtêw,,,
+goes,itohtêw,,,,
+says,itwêw,,,,
+knows,nisitawêyimêw,,,,
+wants it,nitawêyihtam,,,,
+wants him,nitawêyimêw,,,,
+wants her,nitawêyimêw,,,,
+wants them,nitawêyimêw,,,,
+talks,pîkiskwâtêw,,,,
+sees it,wâpahtam,,,,
+sees him,wâpamêw,,,,
+sees her,wâpamêw,,,,
+lives at,wîkiw,,,,
+a little,apisîs,,,,
+listen,mâ,,,,
+miracle,mâmaskâc,,,,
+a while,pitamâ,,,,
+gee,wahwâ,,,,
+gosh,wâcistakâc,,,,
+although,âta,,,,
+very true,tâpwe pikwâni,,,,
+piglet,kohkôsis,,,,
+tipi,mîkiwâhp,,,,
+Cree language,nêhiyawêwin,,,,
+queen,okimâskwêw,,,,
+house,wâskahikan,,,,
+Frenchman,wêmistikôsiw,,,,
+home,wîkiwin,,,,
+English language,âkayâsîmowin,,,,
+hitch,âniskamân,,,,
+diaper,âsiyân,,,,
+feels,itamahcihow,,,,
+learns,kiskinwahamâkosiw,,,,
+feels good,miyomahcihow,,,,
+understands,nisitohtam,nisitohtawêw,,,
+listens,nitohtam,,,,
+speaks Cree,nêhiyawêw,,,,
+tired,nêstosiw,,,,
+hungry,nôhtêhkatêw,,,,
+sleepy,nôhtêhkwasiw,,,,
+very,mistahi,,,,
+hat,astotin,,,,
+shorts,atâmicâsis,kîskicâsis,,,
+scarf,kîsowahpison,,,,
+book,masinahikan,,,,
+kitten,minôsis,,,,
+pony,miscacimosis,,,,
+jacket,miskotâkay,,,,
+river,sîpiy,,,,
+bowl,wiyâkanis,,,,
+dawn,pêtâpan,wâpan,,,
+wheel,wâsakâpayîs,,,,
+small,apisâsin,apisîsisiw,,,
+green,askihtakosiw,askihtakwan,,,
+short,cimisisiw,cimâsin,,,
+high,ispâw,,,,
+fast,kakwêyâhow,,,,
+clean,kanâtan,kanâtisiw,,,
+black,kaskitêsiw,kaskitêwâw,,,
+beautiful,katawasisin,katawasisiw,,,
+tall,kinosiw,kinwâw,,,
+strong,maskawisîw,maskihkîwimâkwan,,,
+red,mihkosiw,mihkwâw,,,
+big,misikitiw,misâw,,,
+nice,miyosiw,,,,
+good,miywâsin,,,,
+ugly,mâyâtan,mâyâtisiw,,,
+orange,osâwisiw,osâwâw,osâwâs,,
+grey,pihkonâkosiw,pihkonâkwan,,,
+broken,pîkopayin,pîkopayiw,,,
+blue,sîpihkosiw,sîpihkwâw,,,
+wine red,watôkimihkosiw,watôkimihkwâw,,,
+likes the taste,wîhkistêw,wîhkistam,,,
+dirty,wiyipâtan,wiyipâtisiw,,,
+pink,wâpikwanîwinâkosiw,wâpikwanîwinâkwan,,,
+white,wâpiskisiw,wâpiskâw,,,
+yellow,wâposâwisiw,wâposâwâw,,,
+delicious,wîhkasin,wîhkitisiw,,,
+purple,wîpi-sîpihkosiw,wîpi-sîpihkwâw,,,
+brown,wîposâwâw,wîpôsâwisiw,,,
+slow,yikicikâwîmakan,yîkicikâwîw,,,
+go away,awas,awasitik,,,
+just kidding,cwâ,,,,
+wait,cêskwa,,,,
+near,cîki,,,,
+that's okay,kiyâm,,,,
+ok,kiyâm,,,,
+always,kâkikê,,,,
+don't,kâya,,,,
+let's,mahti,,,,
+right now,mêkwâc,sêmâk,,,
+here,na,,,,
+because,osâm,,,,
+already,âsay,,,,
+come here,âstam,âstamitik,,,
+okay now,âw,,,,
+never mind,ekosi kîyâm,,,,
+look over there,itâpi nêtê,,,,
+well,êkosi mâka,,,,
+that's it for now,êkosi pita,,,,
+coffee,pihkahcêwâpôs,pihkahtêwâpoy,,,
+meat,wiyâs,,,,
+sit down,apiw,,,,
+has,ayâw,ayâwêw,,,
+looks,itâpiw,kitâpahtam,kitâpamêw,,
+cooks,kîsisam,kîsiswêw,piminawasow,,
+likes,miywêyihtam,,,,
+eats,mowêw,mîciw,,,
+bakes,nawacîw,,,,
+hits,pakamaham,pakamahwêw,,,
+gets up,pasikôw,,,,
+smiles,pâhpisiw,,,,
+hears,pêhtawêw,,,,
+gives a ride,pôsihêw,,,,
+rides,pôsiw,,,,
+blows on,pôtâtam,pôtâtêw,,,
+leaves,sipwêhtêw,nakasiwêw,nakatam,,
```

Comment:
leaves: should probably return nîpiy 'leaf' just in case

Reply:
Good point, but sadly the current thing I’m putting together is only looking at top 3. If you want ‘nîpiy’ you gotta let me know which other one it should replace, perhaps ‘nakasiwêw’?

Reply:
Of those three, yes. I suppose someone might also want the VTA alongside the VTI, but the noun also seems important... will need more thought!